tools/Linux/kodi.sh.in

```diff
@@ -25,16 +25,20 @@ prefix="@prefix@"
 exec_prefix="@exec_prefix@"
 datarootdir="@datarootdir@"
 LIBDIR="@libdir@"
+APP_BINARY=$LIBDIR/${bin_name}/@APP_BINARY@
```

Comment:
This basically means that the default windowing is always the one that was built last, right? Do we want that? Or don't we always want to try wayland (`WAYLAND_DISPLAY` set?) -> x11 (`DISPLAY` set?) -> gbm by default on Linux (also depending on the availability of the binaries of course)?

Reply:
this is the fallback, so it should probably be the one that was built last (as this this the most likely to exist). In the case of our PPA this will be x11 though, since I only install the kodi-wayland binary for the wayland case.
The auto detection is default behavior, but I'll add x11 detection as well. Forgot that

Reply:
OK, with x11 detection in place it should be fine like this